src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -21,49 +111,445 @@ const ActorInit Bg_Spot16_Bombstone_InitVars = {
+    for (index = 0; index < ARRAY_COUNT(D_808B6088); index++) {
+        if (Actor_Spawn(&globalCtx->actorCtx, globalCtx, ACTOR_BG_SPOT16_BOMBSTONE, posRot->pos.x, posRot->pos.y,
+                        posRot->pos.z, 0, 0, 0, D_808B6088[index]) == NULL) {
+            break;
+        }
+    }
+}
+
+void func_808B56BC(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    f32 sinValue;
+    s16 adjustedYawDiff;
+    s32 yawDiff;
+    s32 absYawDiff;
+
+    if (this->actor.xzDistFromLink < 130.0f && this->actor.yDistFromLink < 160.0f &&
+        this->actor.yDistFromLink >= -10.0f) {
+        yawDiff = this->actor.yawTowardsLink - this->actor.shape.rot.y;
+        absYawDiff = ABS(yawDiff);
+
+        adjustedYawDiff = absYawDiff - 0x3FFF;
+
+        if (adjustedYawDiff > 0) {
+            sinValue = Math_Sins(adjustedYawDiff) * this->actor.xzDistFromLink;
+
+            if (sinValue >= 0.0f) {
+                player->actor.posRot.pos.x += sinValue * this->sinRotation;
+                player->actor.posRot.pos.z += sinValue * this->cosRotation;
+            } else {
+                osSyncPrintf("Error 補正出来ない(%s %d)(arg_data 0x%04x)(hosei_angY %x)\n",
+                             "../z_bg_spot16_bombstone.c", 935, this->actor.params, adjustedYawDiff);
+            }
+        }
+    }
+}
+
+void func_808B57E0(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    EnBombf* playerHeldActor;
+    Player* player = PLAYER;
+    EnBombf* currentBomb;
+
+    if (sTimer > 0) {
+        sTimer--;
+    }
+
+    if (sPlayerBomb != NULL) {
+        if (sPlayerBomb->actor.update == NULL) {
+            sPlayerBomb = NULL;
+        } else if (sTimer <= 0 && sPlayerBomb->actor.posRot.pos.y < 1400.0f &&
+                   Math3D_Dist1DSq(sPlayerBomb->actor.posRot.pos.x + 1579.0f,
+                                   sPlayerBomb->actor.posRot.pos.z + 790.0f) < SQ(400.0f) &&
+                   sPlayerBomb->actor.params == 0) {
+            currentBomb = sPlayerBomb;
+            if (currentBomb->timer > 0) {
+                sTimer = currentBomb->timer + 20;
+                func_800800F8(globalCtx, 0x1054, sTimer, NULL, 0);
+            }
+        }
+    } else if (player->stateFlags1 & 0x800) {
+        playerHeldActor = player->heldActor;
+        if (playerHeldActor != NULL && playerHeldActor->actor.type == ACTORTYPE_EXPLOSIVES &&
+            playerHeldActor->actor.id == ACTOR_EN_BOMBF) {
+            sPlayerBomb = playerHeldActor;
+        }
+    }
+}
+
+void func_808B5934(BgSpot16Bombstone* this) {
+    this->actor.draw = BgSpot16Bombstone_Draw;
+    this->actionFunc = func_808B5950;
+}
+
+void func_808B5950(BgSpot16Bombstone* this, GlobalContext* globalCtx) {
+    u8 acFlags;
+
+    func_808B56BC(this, globalCtx);
+    func_808B57E0(this, globalCtx);
+
+    if (globalCtx) {}
+
+    acFlags = this->colliderCylinder.base.acFlags;
+
+    if (acFlags & 2) {
+        this->colliderCylinder.base.acFlags = acFlags & ~2;
```

Comment:
Any chance this works to get rid of the temp?
```suggestion
    if (this->colliderCylinder.base.acFlags & 2) {
        this->colliderCylinder.base.acFlags &= ~2;
```